### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by camrynlindsay**
Hey @NathanOrawi, could you please look over my code? In particular, can you please check my classes contain the right methods suited to the questions in this week's practical?

### CODE REVIEW

**Comment by NathanOrawi** [comment]
Hello @camrynlindsay, your wish is my command. I shall review your practical with maximum effort.

File: prac_09/unreliable_car.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import random
```
Comment (NathanOrawi) [inline prac_09/unreliable_car.py]: In my very short experience, I found it helpful to specify exactly what class I am using from a module. For readability purposes. If I am just importing one or two classes or methods

For short codes it's alright but for long lines of code specifying what exactly you imported helps other coders or future you to get the gist of what's going on when you scan the imports section

File: prac_09/unreliable_car_test.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+"""
+
+from unreliable_car import UnreliableCar
+
+
+def test_unreliable_cars():
+    """Tests for a reliable and unreliable car."""
+    reliable_car = UnreliableCar("Good Car", 100, 90)
+    unreliable_car = UnreliableCar("Bad Bar", 100, 10)
+    for i in range(1, 15):
```
Comment (NathanOrawi) [inline prac_09/unreliable_car_test.py:14]: I Like this, Nice. Very Nice

**Comment by NathanOrawi** [comment]
@camrynlindsay your code for this practical is very concise and clean. It seems that you have a good grasp of the basics.
To my knowledge, I can't find any PEP8 breach or flaws.  🥇

**Comment by camrynlindsay** [comment]
Hey Nathan, I'm sorry to get back to you so late, but thank you for reviewing my prac for week 9!!! 😄

**Pull request merged**